tensorflow/core/kernels/data/dataset_test_base.h

```diff
@@ -52,11 +52,11 @@ class DatasetOpsTestBase : public ::testing::Test {
   static Status ExpectEqual(const Tensor& a, const Tensor& b);
 
   // The method validates whether the two tensor vectors have the same tensors.
-  // If `expect_items_equal` is true, the method will only evaluate the two
+  // If `compare_order` is false, the method will only evaluate the two
```

Comment:
s/evaluate the/evaluate whether the/

Reply:
It is fixed via https://github.com/tensorflow/tensorflow/pull/27969/commits/9ea146a83b5ea9d2d1dceee54ff2994e36b1335b